metabox/metabox/core/machine.py

```diff
@@ -266,12 +260,13 @@ def get_early_setup(self):
         """
 
         commands = [
-            "bash -c 'chmod +x /var/tmp/checkbox-providers/base/bin/*'",
-            "bash -c 'chmod +x /var/tmp/checkbox-providers/resource/bin/*'",
             ("bash -c 'pushd /home/ubuntu/checkbox/checkbox-ng ; "
              "sudo python3 -m pip install -e .'"),
             ("bash -c 'pushd /home/ubuntu/checkbox/checkbox-support ; "
              "sudo python3 -m pip install -e .'"),
+            "bash -c 'sudo mkdir -p /usr/local/share/plainbox-providers-1/'",
+            ("bash -c 'ls /home/ubuntu/checkbox/providers/**/manage.py"
```

Comment:
I like the smartness, but it's not obvious at the first look what it does, and there's not that many of them.
I'm torn here tbh.

Reply:
I have done it this way because if we create a new one it will be hard to understand why it is not present in the metabox container if either me or you don't recall that we have to update the install command

Reply:
Yeah, if you really want to leave it in, then yeah, write a big fat comment there, please.